### ISSUE TITLE
Collaborations

### ISSUE THREAD

**Issue by clayjens** [plexidev/issue-tracker#4]
I'm not entirely sure if this is doable within GitHub, but a feature that acts almost like a hiring board for collaborations. Similar to how this tracker behaves.
The channel that existed within Plexidev was great, but it allowed non-programmers to essentially hire programmers, instead of actually collaborating on a project. That feature brought to here, with an optional tag of _Hire_ or similar, would weed out the non-programmers (because really who who has a GitHub account that doesn't program?).

TL;DR:
- Add a Collaboration & Hiring system to plexidev's GitHub

**Comment by Revolx** [comment]
Currently we do not have plans of this, simply due to it not aligning with our vision or goals for what we want Plexi Development to be. Thought we appreciate the information and ideas of which you have provided it is not something we deem currently as a high priority or requirement for our community.

We may look back over this but we do feel there is other communities and places of which are more active that people can use.

**Comment by clayjens** [comment]
> We may look back over this but we do feel there is other communities and places of which are more active that people can use.

I understand the concerns about what fits the Plexi community, but as it stands now there isn't much of an incentive to join Plexi, besides for help relating to [Captcha Bot](https://top.gg/bot/captcha) and the occasional help to previous YouTube tutorials, the code of which has been discontinued. I feel like Plexi should look for more avenues to grow and curate a community based on many broad subjects and experiences, using a collaboration (and/or hiring) platform as a start for more people to experience the skills of the Developers that Plexi draws in.

**Comment by Zorotic** [comment]
We're slowly working towards the community we want. We may introduce some new features in the future. But at the moment we have are strictly focusing on things such as Server Captcha Bot and it's premium counterparts, and our website.

Please do take this with a grain of salt as I'm incredibly tired right now and as a matter of fact once I finish writing this I'm gonna be going to sleep.
